client/components/Help/Help.js

```diff
@@ -2,37 +2,74 @@ import React, { Component, PropTypes } from 'react';
 import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
 import { connect } from 'react-redux';
 import { NewProblem } from '../NewProblem/NewProblem';
-import { Problem } from './../Problem/Problem';
+import { Problem } from '../Problem/Problem';
+import { AnswersList } from '../AnswersList/AnswersList';
 import { getProblems, addProblem } from '../../actions/actions';
 import './Help.scss';
 
 class Help extends Component {
     state = {
-        isNewProblemVisible: false
+        isNewProblemVisible: false,
+        isAnswersListVisible: false
     };
 
-    componentDidMount() {
-        this.props.dispatch(getProblems());
-    }
-
     newButtonClick = () => {
-        this.setState({
-            isNewProblemVisible: true
-        });
+        if (this.state.isAnswersListVisible) {
+            this.setState({
+                isAnswersListVisible: false
+            });
+            setTimeout(() => {
```

Comment:
try to avoid this timeout with css animations if possible.
most of timeouts are evil!